src/aat/resources/features/F-026/F-026.feature

```diff
@@ -0,0 +1,54 @@
+@F-026
+Feature: F-026: Get case data with UI layout
```

Comment:
In previous func tests suites I worked on and implemented in the past, I've always seen feature folders and files named with a proper business description rather than a numeric code. I find that approach better IMHO. 
I think having features organised in folders like 'crud', 'case creation', 'show condition' etc would be an improvement. But I'm sure this was considered already.

Reply:
I second that and raised it few times before.
AFAIU on top of the above it needs to be kept in sync with a spreadsheet on some confluence page.
As @mario-paniccia says this is confusing to new devs and makes grouping of feature much harder...

Reply:
Is it possible to change that or is it too late?

Reply:
@MSancaktutar I think this is critical. Raise the priority of that ticket